.github/workflows/coverity.yml

```diff
@@ -0,0 +1,49 @@
+name: "Coverity"
+on:
+  schedule:
+    - cron:  '0 1 * * 0'
+
+jobs:
+  analyze:
+    name: Analyze
+    if: github.repository == 'Nordix/hiredis-cluster'
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+
+    - name: Download Coverity
+      run: |
+        cd ..
+        wget -q https://scan.coverity.com/download/linux64 --post-data "token=${TOKEN}&project=hiredis-cluster" -O coverity_tool.tgz
+        mkdir coverity
+        tar xzf coverity_tool.tgz --strip 1 -C coverity
+        echo "$(pwd)/coverity/bin" >> $GITHUB_PATH
+      env:
+        TOKEN: ${{ secrets.COVERITY_TOKEN }}
+
+    - name: Prepare
+      run: |
+        sudo apt-get install -y libevent-dev gcc cmake
+        mkdir build; cd build
+        cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DENABLE_SSL=ON ..
+
+    - name: Build with Coverity
+      run: cov-build --dir cov-int make
+      working-directory: build
+
+    - name: Submit the result to Coverity
+      run: |
+        tar czvf hiredis_cluster.tgz cov-int
+        curl \
+          --form token=${TOKEN} \
+          --form email=andreas@gerstmayr.me \
```

Comment:
@andreasgerstmayr do we need to add your token to the Nordix repo? Or should we use mine or @bjosv's email here instead?

Reply:
Yes, you need to add a GitHub secret named `COVERITY_TOKEN` to the repository. The token is visible here: https://scan.coverity.com/projects/hiredis-cluster?tab=project_settings

I'm not 100% sure what the mail is used for (I only know it doesn't work without), but yes, you can change it to one of you instead. Thanks!

Reply:
* [x] COVERITY_TOKEN added

Should `--form token=${TOKEN}` be changed to `--form token=${COVERITY_TOKEN}`?

Perhaps each build is linked to a person. I don't mind it being you @andreasgerstmayr. I guess it has to be one of us listed here: https://scan.coverity.com/projects/hiredis-cluster?tab=members

Reply:
> Should `--form token=${TOKEN}` be changed to `--form token=${COVERITY_TOKEN}`?

Done.

> Perhaps each build is linked to a person. I don't mind it being you @andreasgerstmayr. I guess it has to be one of us listed here: https://scan.coverity.com/projects/hiredis-cluster?tab=members

Yep, probably - I've updated it to your mail address now, I guess it should be one of the maintainers of this repo :smiley: